integration-test/src/main/java/com/sequenceiq/it/cloudbreak/AwsCheckSpotInstance.java

```diff
@@ -40,18 +41,18 @@ public void checkSpotInstance(Regions region, String hostGroupToCheck, @Optional
         Integer spotInstanceCount = 0;
         IntegrationTestContext itContext = getItContext();
 
-        String stackId = itContext.getContextParam(CloudbreakITContextConstants.STACK_ID);
+        StackV4Response stackResponse = getCloudbreakClient().stackV4Endpoint().get(
+                itContext.getContextParam(CloudbreakITContextConstants.WORKSPACE_ID, Long.class),
+                "", // TODO figure out how to find the name of the stack
```

Comment:
isn't somewhere in the context a param for stack name?

Reply:
dunno, but I my first think is that I don't think so

Reply:
ok